### ISSUE TITLE
Save your spot for #YOLOVISION22 on Tuesday, September 27th!

### ISSUE THREAD

**Issue by glenn-jocher** [ultralytics/yolov5#9589]
Hi everyone!

I'm [Glenn Jocher](https://www.linkedin.com/in/glenn-jocher/), author of [YOLOv5](https://github.com/ultralytics/yolov5) 🚀.

I'd like to invite you to attend the world's first-ever YOLO conference: [#YOLOVISION22](https://ultralytics.com/yolo-vision)!

This virtual event takes place on **September 27th, 2022** with talks from the world's leading vision AI experts from Google, SenseTime's MMLabs, Baidu's PaddlePaddle, Meituan's YOLOv6, Weight & Biases, Roboflow, Neural Magic, OctoML and of course Ultralytics YOLOv5 and many others.

Save your spot at https://ultralytics.com/yolo-vision

Thanks!

<a align="center" href="https://ultralytics.com/yolo-vision" target="_blank">
      <img width="850" src="https://user-images.githubusercontent.com/26833433/192165285-a13a370c-60dc-4bf2-9d63-0573fb8e4d44.jpg"></a>

**Comment by dav1d333** [comment]
Hi @glenn-jocher , are the talks recorded and uploaded somewhere?

**Comment by glenn-jocher** [comment]
@dav1d333 Yes! All #YOLOVISION22 talks are now live at https://youtu.be/Lh9hK-ptMrY

**Comment by glenn-jocher** [comment]
Closing this since the YOLOVISION22 event has passed and there is no remaining repository action here.
